flow/layers/opacity_layer.cc

```diff
@@ -17,15 +19,12 @@ OpacityLayer::~OpacityLayer() {}
 
 #if defined(OS_FUCHSIA)
 
-void OpacityLayer::UpdateScene(SceneUpdateContext& context,
-                               mozart::Node* container) {
-  auto node = mozart::Node::New();
-  node->op = mozart::NodeOp::New();
-  node->op->set_layer(mozart::LayerNodeOp::New());
-  node->op->get_layer()->layer_rect = mozart::RectF::From(paint_bounds());
-  node->op->get_layer()->blend = mozart::Blend::New();
-  node->op->get_layer()->blend->alpha = alpha_;
-  UpdateSceneChildrenInsideNode(context, container, std::move(node));
+void OpacityLayer::UpdateScene(mozart::client::Session& session,
+                               SceneUpdateContext& context,
+                               ContainerNode& container) {
+  mozart::client::OpacityNode node(&session);
```

Comment:
We might want to log when we emit one of these since they aren't supported right now.